CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
       ```
 - [`pkg/test.FrameworkClient`](https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/client.go#L33) methods `List()` and `Delete()` have new signatures corresponding to the homonymous methods of `sigs.k8s.io/controller-runtime/pkg/client.Client`. ([#1876](https://github.com/operator-framework/operator-sdk/pull/1876))
 - CRD file names were previously of the form `<group>_<version>_<kind>_crd.yaml`. Now that CRD manifest `spec.version` is deprecated in favor of `spec.versions`, i.e. multiple versions can be specified in one CRD, CRD file names have the form `<full group>_<resource>_crd.yaml`. `<full group>` is the full group name of your CRD while `<group>` is the last subdomain of `<full group>`, ex. `foo.bar.com` vs `foo`. `<resource>` is the plural lower-case CRD Kind found at `spec.names.plural`. ([#1876](https://github.com/operator-framework/operator-sdk/pull/1876))
+- Upgrade Python version from `2.7` to `3.6`, Ansible version from `2.8.0` to `2.8.5` and ansible-runner from `1.2` to `1.3.4` in the Ansible based images. ([#1947](https://github.com/operator-framework/operator-sdk/pull/1947)) 
```

Comment:
Is there a reason you're choosing python 3.6 when 3.7 is [available](https://docs.travis-ci.com/user/reference/xenial/#python-support)?

Reply:
The libs in the ubi7 image we can try to upgrade it after we are able to use the ubi8 and pass in all CI tests. WDYT? Are you ok with?

Reply:
Makes sense to me